### DISCUSSION TITLE
Teaming up with überdosis on a collaborative editor.

### DISCUSSION THREAD

**Discussion by filipesmedeiros** [toeverything/blocksuite#1005] [Ideas]
Hey y'all! I couldn't help to notice that y'all are building a (great) editor on top of Y.js, and that is also exactly what https://ueberdosis.io/ are doing with Tiptap and Hocuspocus https://tiptap.dev/hocuspocus/introduction.
Do you think, in the spirit of collaboration, it would be good to join forces in some way?

Seems like it follows the _ethos_ of both companies/groups 😄

**Comment by doodlewind** [comment]
Thank you for your interest in our project! We appreciate the impressive contribution of überdosis in the field of collaborative editors, but have not yet received information about further cooperation there. We'll keep our eyes open!
